.circleci/base_config.yml

```diff
@@ -377,6 +418,10 @@ workflows:
           enterprise: true
           publish-artifacts: false
           build-tests: false
+      - run-cppcheck:
+          name: cppcheck
+          requires:
+            - build-ee-x64
```

Comment:
I think this should rather run as part of the lint workflow. I don't see why we need to depend on the build job - we could simply run the cmake configure step ourselves.